tests/main/ubuntu-core-create-user/task.yaml

```diff
@@ -7,6 +7,15 @@ restore: |
     sed -i '/^mvo/d' /var/lib/extrausers/group
     rm -rf /home/mvo
 execute: |
+    if [ "$MANAGED_DEVICE" = "true" ]; then
```

Comment:
When executing the tests on the boards and amd64 instances from images created with ubuntu-image we need first to manually configure the instance through the console-conf process, not sure if that uses the system-user assertion, but after the manual initialization process we have a user there and the device is reported by `snap create-user` as managed.